playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/digitalgoods/GetDetailsCall.java

```diff
@@ -60,7 +64,13 @@ public static GetDetailsCall create(@Nullable Bundle args,
     }
 
     /** Calls the callback provided in the constructor with serialized forms of the parameters. */
-    private void respond(BillingResult code, List<SkuDetails> detailsList) {
+    private void respond(BillingResult result, List<SkuDetails> detailsList) {
+        int responseCode = result.getResponseCode();
+        if (responseCode != BillingClient.BillingResponseCode.OK) {
+            Log.w(TAG, "Billing returned non-OK response code: " + responseCode + ", "
+                    + result.getDebugMessage());
+        }
```

Comment:
I'll probably extract this code out to a different class (and dedupe with `AcknowledgeCall`) in a future PR.